android/app/src/main/res/values-fr/strings.xml

```diff
@@ -763,15 +763,15 @@
 	<!-- App Tip #00 -->
 	<string name="app_tip_00">Merci d\'utiliser nos cartes créées par la communauté !</string>
 	<!-- App tip #01 -->
-	<string name="app_tip_01">Avec vos dons et votre soutien, nous pouvons créer les meilleures cartes du monde !</string>
+	<string name="app_tip_01">Avec vos dons et votre soutien, nous pouvons créer les meilleures cartes du monde !</string>
```

Comment:
Should this go to the last regen commit? You can mark this commit as "edit" during the interactive rebase, and then amend it by removing this change from this commit, and committing it separately. Then squashing it with the following regen commit. Or just remove this change, remove regen commit, finish rebase and regenerate strings again.

Sorry for the hassle, but we need a clean git history, and the skill of editing and amending local commits is required...